std_msgs/Float64 pan_wv
std_msgs/Float64 tilt_wv
std_msgs/Float64 pan_ang
std_msgs/Float64 tilt_ang
std_msgs/Float64 zoom
